src/formats/jsonLd.ts

```diff
@@ -41,30 +47,50 @@ export const getJsonLdParser = (): Parser => {
       onCompleteCallbacks.push(callback);
     },
     parse: async (source, resourceInfo) => {
-      const parser = await getParser(getSourceUrl(resourceInfo));
-      const parserPromise = new Promise<void>((resolve) => {
-        parser.on("data", (quad) => {
-          onQuadCallbacks.forEach((callback) => callback(quad));
-        });
-        parser.on("error", (error) => {
-          onErrorCallbacks.forEach((callback) => callback(error));
-        });
-        parser.on("end", () => {
-          onCompleteCallbacks.forEach((callback) => callback());
-          resolve();
-        });
-        parser.write(source);
-        parser.end();
+      let quads = [] as Quad[];
+      try {
+        const plainQuads = (await jsonld.toRDF(JSON.parse(source), {
+          base: getSourceUrl(resourceInfo),
+        })) as [];
```

Comment:
I didn't know this type annotation was possible. Is this equivalent to `any[]` ?

Reply:
actually, this probably should be `any[]` (ts isn't complaining at me, but for clarity as much as anything)